notebooks/advanced_algorithms/Quantum_Principal_Component_Analysis.ipynb

```diff
@@ -0,0 +1,1514 @@
+{
```

Comment:
<p>"last qubit needs to be"</p><p> </p><p>Would be good to add a little more about how the Braket SDK allows us to write one circuit that can be compiled down to run on various devices.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-braket-algorithm-library/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>